cache-cli/cmd/restore.go

```diff
@@ -27,7 +27,7 @@ var restoreCmd = &cobra.Command{
 
 func RunRestore(cmd *cobra.Command, args []string) {
 	if len(args) > 1 {
```

Comment:
Seems to be incompatible with https://github.com/semaphoreci/toolbox/blob/master/cache#L453 ?

Reply:
Why? The old one is "not zero and not equal to 1", this is "> 1" which means the same thing.

Reply:
You are correct, got confused by the fallback mechanism.